src/world/area_dro/dro_02/npc_merlee.c

```diff
@@ -59,7 +59,9 @@ s8 N(MerleeCoinCosts)[] = {
     50, 20, 5, 0,
 };
 
+#if !VERSION_PAL
 s32 N(pad_XX111)[] = { 0 };
```

Comment:
@ethteck This one should be removable if we'd have an `ALIGNED(8)` on the gfx data. (it's located at `0x####8`)